test/unit/math/rev/err/check_not_nan_test.cpp

```diff
@@ -1,7 +1,55 @@
+#include <stan/math/rev/arr.hpp>
 #include <stan/math/rev/scal.hpp>
 #include <gtest/gtest.h>
+#include <vector>
 #include <limits>
 
+TEST(AgradRevErrorHandlingScalar, CheckNotNanVarCheckVectorized) {
+  using stan::math::check_not_nan;
+  using stan::math::var;
```

Comment:
Declare them outside of test block?

Reply:
I think it would be better to have these declarations more narrow.